components/ui/web/.env.development

```diff
@@ -1,7 +1,7 @@
 NEXT_PUBLIC_FILE_SERVICE_URL=https://apps.sciserver.org/fileservice/api/
 NEXT_PUBLIC_LOGIN_PORTAL_URL=https://apps.sciserver.org/login-portal/
-NEXT_PUBLIC_GRAPHQL_URL=http://localhost:4000/graphql
-# NEXT_PUBLIC_GRAPHQL_URL=https://apps.sciserver.org/graphql
+# NEXT_PUBLIC_GRAPHQL_URL=http://localhost:4000/graphql
+NEXT_PUBLIC_GRAPHQL_URL=https://apps.sciserver.org/graphql
```

Comment:
Can we remove the commented line and then from here on not modify these lines for typical PRs. local workflows should have a way to replace this without committing the changes (either by another file or by restoring after testing)

Reply:
I don't usually modify it, but since I was going to make a change to this file, I decided to commit this one too. For local deployments, it's best to have this point to the apps.sciserver.org version of the GraphQL instead of the local version.